.github/workflows/open-ethereum.yml

```diff
@@ -0,0 +1,245 @@
+name: Open Ethereum
+
+on:
+  push:
```

Comment:
I'm not sure if push is enough, it's not triggered in this PR as you can see as opposed to #11527
I suggest we do
```
on:
  pull_request:
  push:
    branches:
      - master
```